frontend/src/pages/settings/server.vue

```diff
@@ -0,0 +1,255 @@
+<template>
+  <SettingsPage>
+    <template #title>
+      Server Settings
+    </template>
+
+    <template #content>
+      <VCol
+        md="6"
+        class="uno-pb-4 uno-pt-0">
+        <h3 class="mb-2 text-lg font-bold">
+          General Server Settings
+        </h3>
+
+        <VTextField
+          v-model="serverSettings.state.value.name"
+          label="Server Name" />
+
+        <VSelect
+          v-model="serverSettings.state.value.language"
+          :loading="saving"
+          :disabled="saving"
+          variant="outlined"
+          :label="$t('preferredLanguage')"
+          :rules="SomeItemSelectedRule"
+          item-title="Name"
+          item-value="Value"
+          :items="culturesList" />
+
+        <VCheckbox
+          v-model="serverSettings.state.value.quickConnect"
+          label="Enable Quick Connect" />
+
+        <h3 class="text-lg font-bold mt-6 mb-2">
+          Paths
+        </h3>
+
+        <VTextField
+          v-model="serverSettings.state.value.cachePath"
+          label="Cache Path" />
+
+        <VTextField
+          v-model="serverSettings.state.value.metadataPath"
+          label="Metadata Path" />
```

Comment:
I wanted to make a new component which would autocomplete the paths with a file explorer type thing, but thought Id do that in a different PR